packages/ui/react-ui-board/src/Grid/Grid.tsx

```diff
@@ -158,49 +173,65 @@ const RootInner = forwardRef<GridController, RootProps>(
             padding: margin,
```

Comment:
Is the `margin` prop _only_ for styling the padding on this particular element?

Some `Root` subcomponents do render DOM nodes, I’d recommend using a more ordinary pattern, like applying styles by default and letting a `classNames` prop apply overrides.